pkg/config/externalidpconfig.go

```diff
@@ -11,6 +11,12 @@ import (
 const (
 	accessToken              = "accessToken"
 	client                   = "client"
+	clientSecretBasic        = "client_secret_basic"
+	clientSecretPost         = "client_secret_post"
+	clientSecretJwt          = "client_secret_jwt"
+	privateKeyJwt            = "private_key_jwt"
+	tlsClientAuth            = "tls_client_auth"
+	selfSignedTlsClientAuth  = "self_signed_tls_client_auth"
```

Comment:
do these constants and the other constants have to be defined separately?

Reply:
These constants are based on the token authentication types coming from OAuth specification

Reply:
Will use these and remove the ones in oauth package